vtexdocs/devportal
# Fix/documentation cards tooltip

#### What is the purpose of this pull request?

Add tooltip on cards when the card' description is longer than two lines.

#### What problem is this solving?

When the user translates the page (using an extension), the description can be longer than two lines breaking the card.
![image](https://user-images.githubusercontent.com/42784961/177622864-c54fda3c-6da2-439c-b295-617d05893a34.png)

But, by limiting the description to two lines, the user is unable to read its content.
![image](https://user-images.githubusercontent.com/42784961/177623705-4ef0c9cb-293c-4dda-9f8c-17143c000385.png)

So, it's necessary the addition of the tooltip when the description is limited.

![image](https://user-images.githubusercontent.com/42784961/177624035-7106b563-969d-4d77-8986-008e709b423f.png)


#### How should this be manually tested?

Go to this page and try to translate it to any language, check if every time the card description is limited, the tooltip will appears on mouse hover.

#### Types of changes
